src/ui-kit/form-controls/date/date.component.ts

```diff
@@ -543,13 +543,22 @@ export class SamDateComponent
     return `${this.name}_month`;
   }
 
+  monthNameDesc() {
+    return `${this.name}_month Enter Month Here`;
+  }
   dayName() {
     return `${this.name}_day`;
   }
+  dayNameDesc() {
+    return `${this.name}_day Enter Day Here`;
+  }
 
   yearName() {
     return `${this.name}_year`;
   }
+  yearNameDesc() {
```

Comment:
why are you creating new methods, we can use the existing methods to rename the text